databricks/koalas/frame.py

```diff
@@ -86,6 +87,11 @@ def _index_columns(self):
         return [self._sdf.__getitem__(field)
                 for field in self._metadata.index_fields]
 
+    @lazy_property
```

Comment:
@rxin, I think we should just leave it as `property` for now. I think some APIs have `inplace` options and I suspect that won't work with cache.

Reply:
Yea you are right. I got rid of it completely. There's no point doing property since it just shortens a little bit of code.